spec/read_models/received_on_reports/projection_spec.rb

```diff
@@ -48,34 +50,50 @@
         travel_back
       end
 
-      let(:cat1_data) { dataset.fetch(cat1) }
-      let(:extradition_data) { dataset.fetch(extradition) }
+      let(:cat1_initial) { dataset.dig(cat1, initial) }
+      let(:cat1_pse) { dataset.dig(cat1, pse) }
+      let(:extradition_initial) { dataset.dig(extradition, initial) }
+      let(:extradition_pse) { dataset.dig(extradition, pse) }
```

Comment:
How come you have two extradition applications here when you only received one?

Reply:
These lets are for the values in the dataset. So, for example, extradition_pse is zero

Reply:
I've appended _row to the lets to, hopefully, make it clearer.